omnibus_overrides.rb

```diff
@@ -1,6 +1,6 @@
 # Generated by "rake dependencies". Do not edit.
-override :rubygems, version: "2.6.3"
-override :bundler, version: "1.11.2"
+override :rubygems, version: nil
```

Comment:
this just gets the latest definition version? Are these the only 2 that get latest?